ios/fastlane/Fastfile

```diff
@@ -120,12 +120,14 @@ platform :ios do
   lane :promote_to_production do |options|
     latestBuildNumber = latest_testflight_build_number
     # Config in DeliverFile
-    deliver
+    deliver(
```

Comment:
If the version isn't explicitly set when calling deliver then a new version won't be created in Connect.